models/articles.models.js

```diff
@@ -13,4 +14,30 @@ exports.fetchArticles = (articleID)=>{
         return rows
     })
    
+}
+
+exports.fetchArticles = ()=>{
+    
+
+    return db
+    .query(`SELECT  articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url ,COUNT(*) AS comment_count
+    FROM articles
+    INNER JOIN comments ON articles.article_id = comments.article_id
+    GROUP BY articles.article_id
+    ORDER BY articles.created_at DESC;
+    `)
+            
+    .then(({rows})=>{
+
+     
+        
+        /* Strict typecasting the comment_count field so it will be a number and not a string */
+        rows.forEach((article)=>{(article.comment_count) = Number(article.comment_count)})
```

Comment:
You can take care of this in your SQL query.  look at how to make count aggregate function return an INT!